src/components/LoginControl/LoginControl.js

```diff
@@ -12,9 +10,13 @@ export const options = Object.freeze({
       productID: "Silent_Alexa",
       productInstanceAttributes: { deviceSerialNumber: "12345" }
     }
-  }
+  },
+  popup: false
 });
 
+// Redirect URI to handle the response from LoginWithAmazon
+export const redirectUri = "http://localhost:3000/authresponse";
```

Comment:
Do we need a TODO here? This will work locally but isn't it going to break the instance of the app that we host on Github?